pyvista/themes.py

```diff
@@ -2058,6 +2060,26 @@ def save(self, filename):
         with open(filename, 'w') as f:
             json.dump(self.to_dict(), f)
 
+    @property
+    def use_ipyvtk(self):  # pragma: no cover
+        """Set or return the usage of "ipyvtk" as a jupyter backend.
+
+        Deprecated in favor of ``jupyter_backend``.
+        """
+        warnings.warn('use_ipyvtk is deprecated.  Please use '
+                      '``pyvista.jupyter_backend``', DeprecationWarning)
```

Comment:
This should either be `pyvista.global_theme.jupyter_backend` or `pyvista.set_jupyter_backend`. The latter reads better, but it is less informative about the `rcParams` -> `theme` deprecation path.

And the same below.

Reply:
Done.